### PR TITLE
[ODS-5980] Refactor Standard and Extension version optional parameters

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

**Review by simpat-jesus — CHANGES_REQUESTED** [review]
Replace hardcoded extension version by using this job https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/08315e57b7fde8849bf6406e17c1904a3889e471/.github/workflows/Pkg%20EdFi.Ods.Minimal.Template.TPDM.yml#L34

**Comment by semalaiappan** [comment]
@simpat-jesus  Review comments are fixed . Ready for review

**Review by simpat-jesus — APPROVED** [review]

File: .github/workflows/Lib edFi.ods.standard manual.yml [hunk 1]
```diff
@@ -94,17 +94,17 @@ jobs:
     - name: build
       working-directory: ./Ed-Fi-ODS/
       run: |
-        ./build.githubactions.ps1 build -Configuration ${{ env.CONFIGURATION }} -InformationalVersion ${{ env.INFORMATIONAL_VERSION}} -BuildCounter ${{ github.run_number }} -BuildIncrementer ${{env.BUILD_INCREMENTER}}  -ProjectFile "$env:GITHUB_WORKSPACE/Ed-Fi-ODS/Application/EdFi.Ods.Standard/EdFi.Ods.Standard.csproj"   -StandardVersion ${{ matrix.StandardVersion }}
+        .\build.githubactions.ps1 build -Configuration ${{ env.CONFIGURATION }} -InformationalVersion ${{ env.INFORMATIONAL_VERSION}} -BuildCounter ${{ github.run_number }} -BuildIncrementer ${{env.BUILD_INCREMENTER}}  -ProjectFile "$env:GITHUB_WORKSPACE/Ed-Fi-ODS/Application/EdFi.Ods.Standard/EdFi.Ods.Standard.csproj"   -StandardVersion ${{ matrix.StandardVersion }}
```
Comment (stephenfuqua) [inline .github/workflows/Lib edFi.ods.standard manual.yml:97]: Is this going to work on a Linux build agent?

File: .github/workflows/Pkg EdFi.Ods.Minimal.Template.PostgreSQL.yml [hunk 2]
```diff
@@ -49,14 +60,23 @@ jobs:
          echo "StandardVersions=$output" >> $env:GITHUB_OUTPUT
          Write-host "StandardVersions is  $output"
        shell: pwsh
+     - name: Set ExtensionVersions
+       id: Set_ExtensionVersions
+       working-directory: ./Ed-Fi-Extensions/
+       run: |
+         $output = .\build.githubactions.ps1 ExtensionVersions -ProjectFile "$env:GITHUB_WORKSPACE/Ed-Fi-Extensions/Extensions/EdFi.Ods.Extensions.TPDM/EdFi.Ods.Extensions.TPDM.csproj"
+         echo "ExtensionVersions=$output" >> $env:GITHUB_OUTPUT
+         Write-host "ExtensionVersions is  $output"
```
Comment (stephenfuqua) [inline .github/workflows/Pkg EdFi.Ods.Minimal.Template.PostgreSQL.yml:69]: Would it make sense to be consistent here? `echo` and `Write-host` are the same command, so it seems strange to see them both here side by side.

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**